clippy_lints/src/manual_is_ascii_check.rs

```diff
@@ -91,15 +91,16 @@ impl<'tcx> LateLintPass<'tcx> for ManualIsAsciiCheck {
                     CharRange::Digit => Some("is_ascii_digit"),
                     CharRange::Otherwise => None,
                 } {
-                    let mut applicability = Applicability::MaybeIncorrect;
                     let default_snip = "..";
                     // `snippet_with_applicability` may set applicability to `MaybeIncorrect` for
-                    // macro span, so we check applicability manually by comaring `recv` is not default.
+                    // macro span, so we check applicability manually by comparing `recv` is not default.
                     let recv = snippet(cx, recv.span, default_snip);
 
-                    if recv != default_snip {
-                        applicability = Applicability::MachineApplicable;
-                    }
+                    let applicability = if recv == default_snip {
+                        Applicability::MaybeIncorrect
```

Comment:
Hey :wave: just noticed, shouldn't this be [`HasPlaceholders`](https://doc.rust-lang.org/stable/nightly-rustc/rustc_errors/enum.Applicability.html). I missed that during the last review :sweat_smile:

Otherwise LGTM

Reply:
oh, nice catch thanks:pray: